daprdocs/content/en/reference/components-reference/supported-cryptography/kubernetes-secrets.md

```diff
@@ -0,0 +1,39 @@
+---
+type: docs
+title: "Kubernetes Secrets"
+linkTitle: "Kubernetes Secrets"
+description: Detailed information on the Kubernetes secret cryptography component
+---
+
+## Component format
+
+Todo: update component format to correct format for cryptography
+
```

Comment:
Purpose of this component is to load keys that are stored as Kubernetes secrets.

This component uses the built-in Dapr crypto engine, so the note should be added